src/modules/sharing/sharing.controller.ts

```diff
@@ -180,6 +182,18 @@ export class SharingController {
     return this.sharingService.createInvite(user, createInviteDto);
   }
 
+  @UseGuards(ThrottlerGuard)
+  @Get('/invites/:id/validate')
+  @Public()
+  @ApiParam({
+    name: 'id',
+    description: 'Id of the invite to validate',
+    type: String,
+  })
+  async validateInvite(@Param('id') id: SharingInvite['id']) {
```

Comment:
async is not needed here, although you do a try catch. I would advise adding a try-catch with a log.